.idea/laika.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add the whole `.idea` folder to the `.gitignore` as we we all don't use the same IDE.